### ISSUE TITLE
Accessibility: Value Help Dialog

### ISSUE THREAD

**Issue by Sanjay-1312** [SAP/fundamental-ngx#4961]
Accessibility: Value Help Dialog Component (There are couple of accessibility issues found while testing, so i have attached error document for you reference.)
Screenshot:
[Value Help Dialog_Platform_ARIBA Controls.docx](https://github.com/SAP/fundamental-ngx/files/6168951/Value.Help.Dialog_Platform_ARIBA.Controls.docx)

**Comment by Sanjay-1312** [comment]
Issue: 1 fixed

Issue: 2
On navigating to the table using tab key, the entire information available within the table is read at once and a lot of additional information is also read.
![image](https://user-images.githubusercontent.com/78714518/123451611-4a00b500-d5fb-11eb-85a7-1cef5126e97d.png)
![image](https://user-images.githubusercontent.com/78714518/123451632-4f5dff80-d5fb-11eb-8878-192e10fbd3b6.png)
EB:Ideally, on navigating to the table information regarding the table name, along with the row and column count followed by the focused row/column should only be read and no additional information should be read.

issue 3:
Removed from the scope

Issue 4:
![image](https://user-images.githubusercontent.com/78714518/123453994-b2e92c80-d5fd-11eb-90df-e498a88a4b73.png)
OB: The column headers are not identified as headers by the screen reader on navigation. Also, the column headers are not associated to the respective data cells due to which on navigating to the data cell only the value is read and the related column header is not read and additional information should be removed.
EB:Ideally, all the column headers should be identified as headers by the screen reader. Also, the column headers should be associated to the respective data cells.


Issue : 5
Removed from the scope


Issue : 6
Issue still persists
OB: on accessing the tab list, Additional information is read on using the Control+Page down it will switch the browser page. additional information 'Control+Pagedown' should be removed.
![image](https://user-images.githubusercontent.com/78714518/123452197-d14e2880-d5fb-11eb-901b-7d0296a684e3.png)


issue 7:
Removed from the scope

issue 8:
Removed from the scope

Issue 9:
Removed from the scope

Issue: 10:
OB: The text alternative or tooltip information such as 'Back' button and remove button is not provided.
![image](https://user-images.githubusercontent.com/78714518/123452437-09ee0200-d5fc-11eb-9065-bf0528dc97be.png)
![image](https://user-images.githubusercontent.com/78714518/123452509-1a9e7800-d5fc-11eb-8549-bbef257d4dc0.png)

Issue 11:
Removed from the scope

Issue 12:
![image](https://user-images.githubusercontent.com/78714518/123452657-3f92eb00-d5fc-11eb-8a1f-677d0aefc56a.png)
![image](https://user-images.githubusercontent.com/78714518/123453342-0c049080-d5fd-11eb-85bd-c4b1bbb7b18b.png)
OB: On accessing the field, additional information is read, only visible and relevant information should be read.
EB: the label UI type and placeholder value should be read.
OB: The button name and Tooltip information is read. 
EB: the Button name should read and ToolTip information should be removed.

Issue 13:
OB: The issue is partially fixed, the Button name 'Open and Select' is not read.
EB: the  visible button name should be read.

Issue 14:
The issue is still reproducible.

Issue 15:
Fixed

URL: https://deploy-preview-5810--fundamental-ngx.netlify.app/#/platform/value-help-dialog
JAWS 2021
